src/components/atoms/Plugin/IFrame/index.tsx

```diff
@@ -63,11 +64,12 @@ const IFrame: React.ForwardRefRenderFunction<Ref, Props> = (
       frameBorder="no"
       scrolling={autoResize ? undefined : "no"}
       data-testid="iframe"
-      srcDoc=""
+      srcDoc={srcDoc}
       key={html}
       ref={composeRefs(iFrameRef, externalRef)}
       className={className}
       onLoad={onIFrameLoad}
+      sandbox="allow-scripts"
```

Comment:
This may be too strict? I'm doing tests for exist plugins, maybe add some more if needed.